src/scripts/main.js

```diff
@@ -1,3 +1,86 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('.field');
+const appendRow = document.querySelector('.append-row');
+const appendColumn = document.querySelector('.append-column');
+const removeRow = document.querySelector('.remove-row');
+const removeColumn = document.querySelector('.remove-column');
+
+container.addEventListener('click', (e) => {
+  const mainRow = Array.from(table.rows);
+  const tds = mainRow[0].children;
+
+  const item = e.target;
+
+  if (item.classList.contains('append-row')) {
+    if (mainRow.length > 9) {
```

Comment:
you should check it after adding/removing row/column and disable the button immediately